src/components/pod/pod.style.js

```diff
@@ -37,10 +37,12 @@ const StyledBlock = styled.div`
     internalEditButton,
     isHovered,
     isFocused,
+    stretchable,
   }) => css`
     box-sizing: border-box;
     background-color: ${blockBackgrounds(variant, theme)};
     width: 100%;
+    ${stretchable && "height: 100%"};
     ${variant === "tile" && "box-shadow: 0 2px 3px 0 rgba(2, 18, 36, 0.2)"};
     ${noBorder ? "border: none" : `border: 1px solid ${theme.pod.border};`};
```

Comment:
```suggestion
    ${noBorder ? "border: none" : `border: 1px solid ${theme.pod.border}`};
```
Apologies I know it wasn't your change but this will render two `;;` if noBorder is falsy